pkg/yurtadm/constants/constants.go

```diff
@@ -46,8 +47,11 @@ const (
 	EdgeNode  = "edge"
 	CloudNode = "cloud"
 
+	DefaultVIP                   = "10.103.97.2"
```

Comment:
why do we need this VIP?

Reply:
@rambohe-ch 
In the case of multiple masters, yurtadm needs to use VIP to join. Otherwise, which IP should be joined?

Reply:
![image](https://user-images.githubusercontent.com/43909149/180741538-e63bc088-ce05-45d2-b43b-c4c9f59277a2.png)

Reply:
Then kubeadm join can use 10.103.97.12:6443 instead real masters.

Reply:
10.103.97.2 also refer to the source code of `seaer join` .

Reply:
Yurthub already has load balancing capability, it seems that VIP is really not needed. `kubedm join` just needs to specify the IP address of any master, because later, you access apiserver through yurthub.

Reply:
Yurthub already has load balancing capability, it seems that VIP and lvscare are really not needed. kubedm join just needs to specify the IP address of any master, because requests are already made to access the apiserver through yurthub.

I will correct this part of the code.

Reply:
solved.